qt/python/mantidqtinterfaces/mantidqtinterfaces/drill/view/DrillSettingsDialog.py

```diff
@@ -16,6 +16,22 @@
 from mantidqt.widgets.workspaceselector import WorkspaceSelector
 
 
+class MouseScrollEater(QObject):
```

Comment:
Can you please rename this to `MouseScrollEventFilter` instead?

Reply:
Done